### PR TITLE
update readme trying again

### PR DESCRIPTION
**Pull request by dhruvbatra**
Removing redundant information.

### CODE REVIEW

**Review by mathfac — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -91,8 +91,24 @@ Note: only supports Linux; no Windows or MacOS.
...
+    Note #2: Please make sure that you keep your local version of `fairembodied/habitat-challenge:testing_2022_habitat_base_docker` image up to date with the image we have hosted on [dockerhub](https://hub.docker.com/r/fairembodied/habitat-challenge/tags). This can be done by pruning all cached images, using:
+    ```
+    docker system prune -a
+    ```
 
+1. Dataset: First, get access to the Habitat-Matterport3D Dataset scenes by visiting [this link](https://matterport.com/habitat-matterport-3d-research-dataset) and following the given instructions. After getting access to the dataset, carry out the following steps to download the dataset:
+    
+    a) First, you will need to generate a matterport API Token:
+
+<<<<<<< HEAD
```
Comment (mathfac) [inline README.md:110]: @dhruvbatra this should be removed.

File: README.md [hunk 2]
```diff
@@ -102,6 +118,7 @@ Note: only supports Linux; no Windows or MacOS.
     
     a) First, you will need to generate a matterport API Token:
 
+>>>>>>> 35d16d6fbe5b67b6c6faa6e7d520baf2497be8bf
```
Comment (mathfac) [inline README.md:121]: and this.

File: README.md (hunk 1 above)
Comment (dhruvbatra) [reply README.md]: Yeah, apparently I can't git merge. Sigh. Fixed.

File: README.md [hunk 3]
```diff
@@ -162,9 +167,13 @@ We have added a config in `configs/ddppo_objectnav.yaml` that includes a baselin
...
+```
 
-1. **Objectnav**: Download the episodes dataset for HM3D ObjectNav from [link](https://dl.fbaipublicfiles.com/habitat/data/datasets/objectnav/hm3d/v1/objectnav_hm3d_v1.zip) and place it in the folder `habitat-challenge/habitat-challenge-data/data/datasets/objectnav/hm3d`. If placed correctly, you should have the train and val splits at `habitat-challenge/habitat-challenge-data/data/datasets/objectnav/hm3d/v1/train/` and `habitat-challenge/habitat-challenge-data/data/datasets/objectnav/hm3d/v1/val/` respectively. Place HM3D scenes downloaded in step-4 of local-evaluation under the `habitat-challenge/habitat-challenge-data/data/scene_datasets` folder. If you have already downloaded thes files for the habitat-lab repo, you may simply symlink them using `ln -s $PATH_TO_SCENE_DATASETS habitat-challenge-data/data/scene_datasets` (if on OSX or Linux).
+If placed correctly, you should have the train and val splits at `habitat-challenge/habitat-challenge-data/data/datasets/objectnav/hm3d/v1/train/` and `habitat-challenge/habitat-challenge-data/data/datasets/objectnav/hm3d/v1/val/` respectively.
```
Comment (dhruvbatra) [inline README.md:176]: @ykarmesh -- this doesn't appear to be true. More details elsewhere.

Comment (ykarmesh) [reply README.md]: Solved

**Pull request merged**